src/ubuntu/build-scripts/build-install-binfmt-support.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# Stop script on NZEC
+set -e
+# Stop script if unbound variable found (use ${var:-} if intentional)
+set -u
+
+os=$1
+dockerCrossDepsTag="microsoft/dotnet-buildtools-prereqs:${os}-crossdeps"
+
+docker run --privileged --read-only=false --rm $dockerCrossDepsTag apt-get install -y binfmt-support
```

Comment:
Can you explain this?  I don't see how the resulting modification (e.g. installing binfmt-support) is being captured.

Reply:
By marking the image as `--read-only=false` I am assuming the modifications to the image are written back to the original image.

Reply:
Based on the logs, looks like this didn't fix anything.

Reply:
No it is not.  A better approach would be to modify the build-rootfs script to start the container, `docker exec` to add `binfmt-support` and then `docker exec` to run `build-rootfs`